programs/drift/src/math/spot_withdraw.rs

```diff
@@ -26,11 +28,14 @@ pub fn calculate_max_borrow_token_amount(
     borrow_token_twap: u128,
     withdraw_guard_threshold: u128,
 ) -> DriftResult<u128> {
+    // maximum borrows permitted for an immediate withdraw
+    // allows at least up to the withdraw_guard_threshold
+    // and between ~15-80% utilization with friction on twap in 10% increments
     let max_borrow_token = withdraw_guard_threshold.max(
         (deposit_token_amount / 6)
```

Comment:
how useful is this case? until the `deposit_token_amount` is 6x larger than withdraw_guard_threshold, wont be hit. once that's hit, can still instantly increase borrows by `deposit_token_amount / 10`, which is pretty close to `deposit_token_amount` / 6 even if `borrow_token_twap` is 0.

Reply:
feel like this can be merged with the utilization check?

i'm this as:
1) below the withdraw guard threshold, you can go to 100% utilization
2) utilization can immediately be 16%
3) utilization can go up by ~10%/hour
4) max utilization is 80%

Reply:
by using `borrow_token_twap` you can prevent the amount of borrow tokens for going too high too fast, even if deposits increase abruptly

Reply:
if there is a bridged asset which goes to zero, but oracle doesnt reflect it:

attacker can keep depositing USDC to immediately borrow as much USDC as they want. once dust settles, they can slowly take out their USDC deposits, so long as they act faster than others to get back to 100% utilization.